### ISSUE TITLE
Create a Policy that request an access token to an Idp(AAD) so it can be later used for calling a BackEnd

### ISSUE THREAD

**Issue by davihern** [dsanchor/apim-demo-infra#28]

**Comment by dsanchor** [comment]
Is that one possible to be implemented with https://learn.microsoft.com/en-us/azure/api-management/authorizations-overview?

**Comment by davihern** [comment]
Yes, that is the way to do it. A policy is needed to call that authorization context and set the Authorization header: https://learn.microsoft.com/en-us/azure/api-management/authorizations-how-to#step-3-authorize-with-github-and-configure-access-policies
